packages/bpk-component-navigation-bar/src/BpkNavigationBar.module.scss

```diff
@@ -85,8 +89,14 @@
   }
 
   &__trailing-item {
+    display: flex;
     grid-column: 3;
     justify-self: end;
+    justify-content: center;
+
+    &--icon {
+      width: tokens.bpk-spacing-lg();
```

Comment:
In figma the icon is expected to have a "Hug" of 24px Padding increasing the icon area from 16 to 24px. Though the hug is not expected on a Link that is passed.